src/app/types/self-subject-access-review.ts

```diff
@@ -0,0 +1,20 @@
+export class SelfSubjectAccessReview {
+  readonly kind = 'SelfSubjectAccessReview';
+  readonly apiVersion = 'authorization.k8s.io/v1';
+  readonly spec = {
+    resourceAttributes: {
+      namespace: 'default',
+      verb: '',
+      resource: '',
+    },
```

Comment:
I think the group is missing in the request
```suggestion
    resourceAttributes: {
      namespace: 'default',
      verb: '',
      resource: '',
      group: 'appuio.io',
    },
```